src/main/resources/META-INF/plugin.xml

```diff
@@ -211,6 +211,21 @@
             <!-- In $default keymap `control` key (Windows/Linux) corresponds to `cmd` key on macOS -->
             <keyboard-shortcut keymap="$default" first-keystroke="control alt shift M" />
         </action>
+
+        <action id="GitMachete.CheckoutNextAction"
+                class="com.virtuslab.gitmachete.frontend.actions.navigation.CheckoutNextAction">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt DOWN" />
```

Comment:
Also add `alt RIGHT` for this action? coz we don't want to end up in a situation where alt-right does something completely unrelated to Git Machete when Git tool window is focused